linchpin/tests/mockdata/dummy/fetch/ws2/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
So I removed ws2 from fetch and replaced it with a smaller directory. I also removed the tar files from ws1, can I commit and push what I have or do you want me to rewrite ws1 as well?